strongbox-web-core/src/test/resources/application-test.yaml

```diff
@@ -1,3 +1,8 @@
 strongbox:
   basedir: ./target
   url: http://${strongbox.host:localhost}:${strongbox.port:48080}
+
+management:
+  health:
+    ldap:
```

Comment:
why this `ldap` needed? :)

Reply:
workaround for https://github.com/spring-projects/spring-ldap/issues/473

we're using `spring-ldap-core-2.3.2.RELEASE`. It's fixed in `2.3.3`

Reply:
can you please leave a `TODO` here as well?